v3/integration/lints/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import "testing"
+
+func TestRun(t *testing.T) {
```

Comment:
Can you add a comment about what this is testing? I'm not 100% sure I follow.

Reply:
lol it's testing `run` which is a top-ish level function for running the whole suite on a directory recursively.

So you get `TestRun` which tests `run`, but when you read it you're like, "yeah, it's a test run...of what?"

I'll come up with something better.